features/storage/FEATURE_STORAGE/TESTS/cfstore/add_del/add_del.cpp

```diff
@@ -60,8 +60,8 @@ UVISOR_SET_MODE_ACL(UVISOR_ENABLED, cfstore_acl_uvisor_box_add_del_g);
 #endif /* YOTTA_CFG_CFSTORE_UVISOR */
 
 static cfstore_kv_data_t cfstore_add_del_test_07_data[] = {
-        CFSTORE_INIT_1_TABLE_MID_NODE,
-        { NULL, NULL},
+    CFSTORE_INIT_1_TABLE_MID_NODE,
+    { NULL, NULL},
```

Comment:
Not sure what you require here. Astyle left this untouched. Changing to {NULL, NULL}. Hope this is as intended.